Cargo.toml

```diff
@@ -31,8 +31,9 @@ version = "0.2"
 path = "./dotrix_ui"
 optional = true
 
-[dev-dependencies.cgmath]
-version = "0.17"
+[dependencies.dotrix_math]
```

Comment:
I believe we don't need it here. `dotrix_math` must be a dependency of `dotrix_core` and we should re-export it inside. It is different to `dotrix_ui` because it is an optional dependency which is not used by core.